ci/publish-solana-tar.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+
+cd "$(dirname "$0")/.."
+
+DRYRUN=
+if [[ -z $BUILDKITE_BRANCH ]] || ./ci/is-pr.sh; then
```

Comment:
we probably don't need this check.  This script isn't run for PRs ever and it'll fail at the end without the aws credentials loaded anyway

Reply:
BUILDKITE_BRANCH check is valid though, right?

Reply:
I'd drop that check too actually.   CHANNEL will be empty if BUILDKITE_BRANCH is missing and we'll still bail

Reply:
So, no DRYRUN either?